### ISSUE TITLE
Wrong power measurements after updating deCONZ to 6.16.0

### ISSUE THREAD

**Issue by erik3rik** [home-assistant/addons#2679]
### Describe the issue you are experiencing

I have several power measurement devices using different integrations and protocols.

After updating deCONZ add-on from 6.15.0 to 6.16.0 (i.e. bump underlying deCONZ from 2.17.1 to 2.18.2), all of my Aqara "smart plug" power measurement plugs reports 10x the power usage compared to before. If it was 60 watts before, it is now 600.

The total energy used (measured in kWh is unchanged, though). 

Example: Powering a single device with a known  power draw of 60W is now reported as 600W in home assistant. But after one hour, the kWh sensor is only increased with 0.060 kWh rather than 0.600 kWh. That leads me to believe there is a simple decimal point error in the "watt sensor" reported value and no underlying error in the measurement. This happened to all 5 of the Aqara Smart Plugs connected via ZigBee through the deCONZ-plugin at the same time, exactly after the version upgrade.

Restarting the deCONZ-add-on fixes nothing, nor restarting the entire hardware with the OS + everything else.

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

deCONZ

### What is the version of the add-on?

6.16.0

### Steps to reproduce the issue

1. Run on deCONZ 6.15.0
2. Use an Aqara Smart Plug and connect something with a fairly even power consumption
3. Display a lovelace graph of the power consumption (measured in watts) for the said plug
4. Update deCONZ to 6.16.0 
5. See that all power values from now on is suddenly 10x what they were before
...

### Anything in the Supervisor logs that might be useful for us?

```txt
Nothing in the logs.
```

### Anything in the add-on logs that might be useful for us?

```txt
Nothing (relevant) in the add-on logs.
```

### Additional information

![Screenshot 2022-09-24 at 11 48 10](https://user-images.githubusercontent.com/46067475/192091627-465db750-53cd-4923-bf96-832598e2bbf5.png)

**Comment by amund7** [comment]
I have the same issue (with Aquara smart plugs), tried same steps as you, rollback to 6.15.0 fixes the problem. Following

**Comment by Mask3007** [comment]
Had the same issue and updated firmware like mentioned here:
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6342

No I have other issues about power measurements

**Comment by carentius** [comment]
I'm joining the thread starter. All my Aqara Smart Plugs measuring with a factor of 10.
Rolling back to 6.15.0 resolve the issue.

**Comment by erik3rik** [comment]
Just to clarify, rolling back to 6.15.0 solves the issue for me too. So there is nothing wrong with the plug itself or Home Assistant in general, nor the ZigBee network. The newer 6.16.0 simply interpret the values coming from the plugs differently than the older 6.15.0 did, and the older 6.15.0 was doing it the right way...

**Comment by ondrackajan** [comment]
I have the same issue with Xiaomi smart plugs. Reverting to 6.15.0 solved the issue so the problem must be with 6.16.0.

**Comment by thorsten** [comment]
I have the same issue as well, reverting to 6.15.0 fixed it.

**Comment by Carxos** [comment]
I have the same issue that the power reports 10x the actual value. It started after updating to 6.16.0.

**Comment by ludeeus** [comment]
This needs to be fixed in deconz, please report the issue to them 👍 
The addon only wrap the software.

**Comment by DanDixon** [comment]
I just started having this problem with the power measurements being 10x what they should be on Oct 22 (not even sure what changed, maybe updating to Home Assistant 2022.10.5)

How do you roll back to 6.15.0?

**Comment by carentius** [comment]
Aften updating my smartplugs it’s working correct again.
[https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Xiaomi-smart-plugs-not-working-correctly](url)

**Comment by EG-ITz** [comment]
Hoping to save time to other people:
I had the same issue upgrading deconz (10x measurement) and I upgrade Aqara firmware as suggested.
After firmware upgrade, power indicator in Home Assistant updates with a big delay and often with 0W value.
I found the problem; after upgrading Aqara firmware, right power value are exposed to a new sensor (Power 38).
I never realized it before because on my dashboard I exposed only the old sensor (dryer power).
![image](https://github.com/home-assistant/addons/assets/14575173/4c054b00-2ce3-45b5-a366-544da60a0933)
![image](https://github.com/home-assistant/addons/assets/14575173/af7e7ff7-d929-4d2d-970e-c58423556c6d)
